lib/src/validationsDispatcher.ts

```diff
@@ -59,20 +61,21 @@ export class ValidationDispatcher {
   }
 
   fireAllFieldsValidations(
-    vm: Object,
+    vm: any,
+    vmKeys: string[],
```

Comment:
Why do you need to pass `vmKeys` as parameter instead of create vmKeys inside this method?